Fix: typos in first half of ..\docs\scarpet\api (#275)
gnembon/fabric-carpet: docs/scarpet/api/BlocksAndWorldAccess.md

### `biome(pos)`

Returns the biome at that block position.

### `solid(pos)`

Boolean function, true if the block is solid.

### `air(pos)`

> Boolean function, true if a block is air, cave air, or void air, or any other air they come up with.

Reviewer: ```suggestion
Boolean function, true if a block is air... or cave air... or void air... or any other air they come up with.
```

### `loaded_status(pos)`

> Returns loaded status as per new 1.14 chunk ticket system, 0 for inaccessible, 1 for border chunk, 2 for (non-entity) ticking, 

Reviewer: ```suggestion
Returns loaded status as per new 1.14 chunk ticket system, 0 for inaccessible, 1 for border chunk, 2 for redstone ticking, 
```

Checks wordgen eligibility for a structure in a given chunk. If no structure is given, or `null`, then it will check
 for all structures. If bounding box of the structures is also requested, it will compute size of potential
  structures. This function, unlike other in the `structure*` category is not using world data nor accesses chunks

>   making it preferred for scoping ungenerated terrain, but it takes some compute power to calculate the structure.

Reviewer: ```suggestion
  making it preferred for scoping ungenerated terrain, but it takes some compute resources to calculate the structure.
```
